packages/bot-web-ui/src/pages/bot-builder/quick-strategy/inputs/qs-input.tsx

```diff
@@ -81,7 +81,7 @@ const QSInput: React.FC<TQSInput> = observer(
                                 <Popover
                                     alignment='bottom'
                                     message={error}
-                                    is_open={is_mobile ? !!error : !!error && has_focus}
+                                    is_open={is_mobile ? !!error : has_focus && !!error}
```

Comment:
@maryia-matskevich-deriv any specific reason to change it on this test case?

Reply:
we can remove it, thanks for you noticed👌

Reply:
removed